core/vm/runtime/runtime.go

```diff
@@ -93,6 +94,9 @@ func setDefaults(cfg *Config) {
 			return common.BytesToHash(crypto.Keccak256([]byte(new(big.Int).SetUint64(n).String())))
 		}
 	}
+	if cfg.BaseFee == nil {
+		cfg.BaseFee = new(big.Int)
```

Comment:
Shouldn't this be `params.InitialBaseFee`? Also, shouldn't we aso set `LondonBlock` in the chain confg above?